bugs/gh-204.txt

```diff
@@ -0,0 +1,1151 @@
+users:
+  - name: testcloudinit
+    sudo: false
+chpasswd:
+  expire: false
+  list:
+    - testcloudinit:RANDOM'
+Creating test-focal-proposed
++ lxc exec test-focal-proposed -- cloud-init status --wait --long
+................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
+status: done
+time: Wed, 17 Jun 2020 14:12:21 +0000
+detail:
+DataSourceNoCloud [seed=/var/lib/cloud/seed/nocloud-net][dsmode=net]
++ verify_user test-focal-proposed focal
++ name=test-focal-proposed
++ series=focal
++ '[' focal = xenial ']'
++ expected_output=testcloudinit:x:1000:1000::/home/testcloudinit:/bin/sh
+++ lxc exec test-focal-proposed -- getent passwd
+++ grep testcloudinit
++ actual_output=testcloudinit:x:1000:1000::/home/testcloudinit:/bin/sh
++ '[' testcloudinit:x:1000:1000::/home/testcloudinit:/bin/sh = testcloudinit:x:1000:1000::/home/testcloudinit:/bin/sh ']'
++ echo 'SUCCESS: focal created the right user'
+SUCCESS: focal created the right user
++ verify_password_change test-focal-proposed focal
++ name=test-focal-proposed
++ series=focal
++ expected_output=1
+++ lxc exec test-focal-proposed -- cat /var/log/cloud-init.log
+++ grep 'Changing password for \['\''testcloudinit'\''\]'
+++ wc -l
++ actual_output=1
++ '[' 1 = 1 ']'
++ echo 'SUCCESS: focal change password for testcloudinit'
+SUCCESS: focal change password for testcloudinit
++ get_boot_load_time test-focal-proposed
++ name=test-focal-proposed
++ lxc exec test-focal-proposed -- systemd-analyze
+Startup finished in 5min 34.563s (userspace) 
+graphical.target reached after 5min 33.321s in userspace
++ lxc exec test-focal-proposed -- systemd-analyze blame
+5min 28.374s snapd.seeded.service                
```

Comment:
Well that's a bit scary for a boot time. can we re-run and see if you get a comparable startup cost?